src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js

```diff
@@ -146,8 +147,15 @@ Actions = {
 		Actions.blurWindow();
 
 		async.waterfall([
-			Actions.askIfUserWantsToSave,
```

Comment:
@carterbs let's play chess and not checkers. We know that customers are going to ask us to go the other direction (promptOnDirty), so go ahead and put a conditional in place, if(promptOnDirty), so that we have a pathway for both strategies. Later on we will tie that to a preference or config. What we don't want to lose is the logic to handle the prompting and then have to reimplement it later.